CMakeLists.txt

```diff
@@ -14,14 +14,46 @@ project(
 include(cmake/project-is-top-level.cmake)
 include(cmake/variables.cmake)
 
+find_package(PkgConfig REQUIRED)
+
+find_package(pcre QUIET)
+
+if (NOT pcre_FOUND)
+    # vcpkg provides pcre cmake config as unofficial-pcre
+    find_package(unofficial-pcre CONFIG REQUIRED)
+    add_library(pcre::libpcre ALIAS unofficial::pcre::pcre)
+endif ()
+
+find_package(pcre2 QUIET)
+
+if (NOT pcre2_FOUND)
+    pkg_check_modules(pcre2 REQUIRED libpcre2-8 IMPORTED_TARGET)
```

Comment:
makes lnav compatible with vcpkg pcre2 package. it does not ship cmake config but ships pkg-config which is used here